packages/itwinui-css/src/button/button-group.scss

```diff
@@ -27,8 +27,6 @@
     }
 
     #{$childZindexSelector} {
-      position: relative;
```

Comment:
why removed? i think this might break some z-index handling

Reply:
Added back